blocks/libs/test/qa_KnownSharedLibBlocks.cpp

```diff
@@ -31,54 +31,55 @@ const boost::ut::suite TagTests = [] {
-        expect(registry.isBlockKnown("gr::testing::SlowSource<complex<float32>>"sv));
-        expect(registry.isBlockKnown("gr::testing::SlowSource<gr::Packet<float32>>"sv));
-        expect(registry.isBlockKnown("gr::testing::SlowSource<gr::Tensor<float32>>"sv));
-        expect(registry.isBlockKnown("gr::testing::SlowSource<gr::DataSet<float32>>"sv));
-        expect(registry.isBlockKnown("gr::testing::CountingSource<float32>"sv));
-        expect(registry.isBlockKnown("gr::testing::CountingSource<complex<float32>>"sv));
-        expect(registry.isBlockKnown("gr::testing::Copy<float32>"sv));
-        expect(registry.isBlockKnown("gr::testing::Copy<complex<float32>>"sv));
-        expect(registry.isBlockKnown("gr::testing::Copy<gr::Packet<float32>>"sv));
-        expect(registry.isBlockKnown("gr::testing::Copy<gr::Tensor<float32>>"sv));
-        expect(registry.isBlockKnown("gr::testing::Copy<gr::DataSet<float32>>"sv));
-        expect(registry.isBlockKnown("gr::testing::HeadBlock<float32>"sv));
-        expect(registry.isBlockKnown("gr::testing::HeadBlock<complex<float32>>"sv));
-        expect(registry.isBlockKnown("gr::testing::HeadBlock<gr::Packet<float32>>"sv));
-        expect(registry.isBlockKnown("gr::testing::HeadBlock<gr::Tensor<float32>>"sv));
-        expect(registry.isBlockKnown("gr::testing::HeadBlock<gr::DataSet<float32>>"sv));
-        expect(registry.isBlockKnown("gr::testing::NullSink<float32>"sv));
-        expect(registry.isBlockKnown("gr::testing::NullSink<complex<float32>>"sv));
-        expect(registry.isBlockKnown("gr::testing::NullSink<gr::Packet<float32>>"sv));
-        expect(registry.isBlockKnown("gr::testing::NullSink<gr::Tensor<float32>>"sv));
-        expect(registry.isBlockKnown("gr::testing::NullSink<gr::DataSet<float32>>"sv));
-        expect(registry.isBlockKnown("gr::blocks::fileio::BasicFileSink<float32>"sv));
-        expect(registry.isBlockKnown("gr::blocks::type::converter::Convert<float32, float32>"sv));
-        expect(registry.isBlockKnown("gr::blocks::type::converter::Convert<float32, float64>"sv));
-        expect(registry.isBlockKnown("gr::blocks::type::converter::ScalingConvert<float32, float32>"sv));
-        expect(registry.isBlockKnown("gr::blocks::type::converter::ScalingConvert<float32, float64>"sv));
-        expect(registry.isBlockKnown("gr::basic::DataSink<float32>"sv));
-        expect(registry.isBlockKnown("gr::blocks::basic::SchmittTrigger<float32, (gr::trigger::InterpolationMethod)0>"sv));
-        expect(registry.isBlockKnown("gr::electrical::PowerMetrics<float32, 3ul>"sv));
-        expect(registry.isBlockKnown("gr::http::HttpBlock<float32>"sv));
-        expect(registry.isBlockKnown("gr::filter::fir_filter<float32>"sv));
-        expect(registry.isBlockKnown("gr::blocks::fft::FFT<float32>"sv));
+    "CheckAvailableBlocks"_test = [&] {
+        expect(gt(registry.availableBlocks().size(), 20UZ));
+
+        expect(registry.isBlockAvailable("gr::basic::ClockSource"sv));
```

Comment:
what about calling it simply `isAvailable(std::string_view blockName)`? More terse and the method/parameter name makes it self-explanatory.